src/parser/utils.ts

```diff
@@ -143,3 +143,97 @@ export function writeToBytes(
     bytes.length,
   );
 }
+
+export function utf8Slice(buf: Uint8Array, start: number, end: number) {
```

Comment:
Can we replace this function with something native?

Reply:
Yes it was only some hacky stuff

Reply:
So what is the alternative?

Reply:
I don't know I think I just yoinked it from the nodejs buffer Compact layer